scripts/travisci/check_flake8.sh

```diff
@@ -89,21 +83,37 @@ E731 # do not assign a lambda expression, use a def
 ignored_errors_incremental=(
 )
 
+# DOCSTRING
+# The following error code enables all the error codes for docstring defined
+# here:
+# http://pep257.readthedocs.io/en/latest/error_codes.html
+# If one of the errors needs to be ignored, just add it to the ignore array.
+errors_incremental_doc="${errors_incremental[@]} D"
```

Comment:
let's call these "errors_add" and "errors_change" to be clear

Reply:
Done